server/runtime/src/main/java/org/infinispan/server/logging/Log.java

```diff
@@ -132,4 +132,8 @@
 
    @Message(value = "Cannot find a network address which matches the supplied configuration", id = 80033)
    CacheConfigurationException invalidNetworkConfiguration();
+
+   @LogMessage(level = Logger.Level.INFO)
+   @Message(value = "Server listening on %s://%s:%d", id = 80034)
```

Comment:
Do we have a server name we could add to the log?

Reply:
We could use the node name. Is that what you mean ?

`Server 'fluffy' listening on https://localhost:11222`

Reply:
Done

Reply:
Yeah, looks good!